### ISSUE TITLE
Clarify transcript requirements

### ISSUE THREAD

**Issue by LJWatson** [alphagov/accessibility-guidance#24]
In [Understanding WCAG](https://github.com/alphagov/accessibility-guidance/wiki/Understanding-WCAG-2.0) it says of Guideline 1.2:
"a transcript for any video with sound, or an audio description of the video ("

In this instance the transcript is for the benefit of people unable to see the video, rather than those unable to hear it. Suggest:
a transcript containing visual descriptions , or an audio description of the video )

**Comment by spencerball** [comment]
Thanks Leonie. It wasn't immediately clear to me what you meant by "visual descriptions" so I had another look at the WCAG guideline.

Is this a better description that what I'd originally drafted?

(You need to provide...) a transcript describing the video, including what people are doing and saying, and what’s happening during any gaps in sound - or an audio description of the video

**Comment by LJWatson** [comment]
This SC is intended to benefit blind and partially sighted people, so the emphasis needs to be on the transcript providing visual descriptions - as well as the spoken dialogue required by other SC. It's the audio description that needs to slot into the gaps of the original soundtrack - a transcript is a separate text document and so timing is irrelevant.

Try this:
"You need to provide a transcript that describes what's happening in the video (as well as what people are saying), or an audio described version of the video."

**Comment by spencerball** [comment]
That's perfect. Thanks Leonie.
